CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,61 @@
+# Contributor Covenant Code of Conduct
+
+## Overview
+
+Define the code of conduct followed and enforced for all RAPIDS projects.
```

Comment:
We should probably change the RAPIDS reference to... whatever Bryce thinks is appropriate and will work for all the projects we may want to add this to.

Reply:
Done